jobs/nightly-samba-rpm-builds.yml

```diff
@@ -0,0 +1,60 @@
+- job:
+    name: gluster_nightly-samba-rpm-builds
+    node: gluster
+    description: Build Samba RPMs.
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: https://github.com/gluster/centosci.git
+        branches:
+        - origin/master
+
+    properties:
+    - github:
+        # TODO: change the repo to final location
+        url: https://github.com/obnoxxx/samba-integration
```

Comment:
What is the purpose of this github property?

Reply:
Ah, I should comment it out like the following block for github PR triggers ;-)

(The repo contains the current samba build and is triggered in the nightly samba build instructions, and it is cloned and invoked in the nightly-samba-builds.sh script from the job.)

Reply:
commented out